specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/stable/2024-05-01/appconfiguration.json

```diff
@@ -0,0 +1,2890 @@
+        },
+        "dataPlaneProxy": {
+          "$ref": "#/definitions/DataPlaneProxyProperties",
+          "description": "Property specifying the configuration of data plane proxy for Azure Resource Manager (ARM)."
+        },
+        "createMode": {
+          "type": "string",
+          "description": "Indicates whether the configuration store need to be recovered.",
+          "enum": [
+            "Recover",
+            "Default"
+          ],
+          "x-ms-enum": {
+            "name": "CreateMode",
+            "modelAsString": false
+          },
+          "x-ms-mutability": [
+            "create"
+          ]
+        }
+      }
+    },
+    "EncryptionProperties": {
+      "type": "object",
+      "description": "The encryption settings for a configuration store.",
+      "properties": {
+        "keyVaultProperties": {
+          "$ref": "#/definitions/KeyVaultProperties",
+          "description": "Key vault properties."
+        }
+      }
+    },
+    "PrivateEndpointConnectionReference": {
+      "type": "object",
+      "description": "A reference to a related private endpoint connection.",
+      "properties": {
+        "id": {
+          "description": "The resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "The name of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "The type of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "$ref": "#/definitions/PrivateEndpointConnectionProperties",
+          "description": "The properties of a private endpoint connection.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "DataPlaneProxyProperties": {
+      "type": "object",
+      "description": "The data plane proxy settings for a configuration store.",
+      "properties": {
+        "authenticationMode": {
+          "description": "The data plane proxy authentication mode. This property manages the authentication mode of request to the data plane resources.",
+          "enum": [
+            "Local",
+            "Pass-through"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "AuthenticationMode",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Local",
+                "description": "The local authentication mode. Users are not required to have data plane permissions if local authentication is not disabled."
+              },
+              {
+                "value": "Pass-through",
+                "description": "The pass-through authentication mode. User identity will be passed through from ARM, requiring user to have data plane action permissions (Available via App Configuration Data Owner/ App Configuration Data Reader)."
```

Comment:
>ARM [](http://example.com/codeflow?start=112&length=3)

in descriptions you should expand the acronyms

Reply:
Updated to Azure Resource Manager (ARM)